### PR TITLE
Address more items from the review

### PR DESCRIPTION
**Pull request by tomrittervg**

### CODE REVIEW

**Review by ln5 — COMMENTED** [review]
This change doesn't seem to address the question of whether a final STH is a verifiable source.

**Review by ln5 — APPROVED** [review]

File: draft-ietf-trans-gossip.md [hunk 1]
```diff
@@ -894,21 +894,18 @@ These configuration needs, and the simple fact that it would require
...
 that did not implement the feature would open itself and its users to
 attack without any possibility of detection.
 
-If SCT Feedback is not deployed by a webserver, malicious logs will be
-able to attack all users of the webserver (who do not have a Trusted
-Auditor relationship) with impunity. Additionally, users who wish to
-have the strongest measure of privacy protection (by disabling STH
-Pollination Proof Fetching and forgoing a Trusted Auditor) could be
-attacked without risk of detection.
+If SCT Feedback is not deployed by a webserver, a webserver may never
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: Suggested changes in wording:
- s/a webserver/that webserver/1 
or
- "A webserver not deploying SCT Feedback may never learn that it was a target [...]"; s/to be clear//1

File: draft-ietf-trans-gossip.md [hunk 2]
```diff
@@ -1019,15 +1016,17 @@ are those who do not have a Trusted Auditor relationship.
...
+out an attack in at least two ways:
 
 In the first attack, the log can present a split view of the log for
 all time. The only way to detect this attack is to resolve each view
 of the log to the two most recent STHs and then force the log to
 present a consistency proof. (Which it cannot.) This attack can be
 detected by CT auditors participating in STH Pollination, as long as
 they are explicitly built to handle the situation of a log
-continuously presenting a split view.
+continuously presenting a split view. We highly recommend they
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: Isn't language like "we highly recommend" more the language of a report?
Could we phrase it more in terms of MAY, SHOULD, MUST?

File: draft-ietf-trans-gossip.md [hunk 3]
```diff
@@ -1039,6 +1038,13 @@ regard the response as a transient error. This attack can be detected
 using SCT Feedback, or an Auditor of Last Resort, as presented in
 {{blocking-policy-response}}.
 
+Both of these attack variants can be detected by CT auditors who have
+obtained a STH of a 'abnormal' view of the log. However, they may not
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: s/a/an/g

File: draft-ietf-trans-gossip.md [hunk 4]
```diff
@@ -984,19 +984,22 @@ auditor.
...
 
-The interactions of the mechanisms is thus outlined:
+Assuming no other log consistency measures exist, clients who perform
+only a subset of the mechanisms described in this document are exposed
+to the following vulnerabilities:
 
 HTTPS clients can be attacked without risk of detection if they do not
-participate in any of the three mechanisms.
+participate in any of the three mechanisms (or any other, unspecified
+mechanisms).
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: _Any_ other is not technically true. I suggest removing the part within parentheses -- that type of "future proofing" seems unnecessary.

File: draft-ietf-trans-gossip.md [hunk 5]
```diff
@@ -1019,14 +1022,14 @@ trusted log that has actively decided to be malicious. It can carry
...
-present a consistency proof. (Which it cannot.) This attack can be
-detected by CT auditors participating in STH Pollination, as long as
-they are explicitly built to handle the situation of a log
-continuously presenting a split view. We highly recommend they
-detect the case where two or more STHs cannot be resolved via
-consistency proofs for some period of time.
+all time. This attack can be detected by CT auditors, but a naive
+auditor implementation may fail to do so. The simplest, least
+efficient way to detect the attack is to mirror the entire log and
+assert inclusion of every peice of data. If an auditor does not mirror
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: s/peice/piece/1

File: draft-ietf-trans-gossip.md [hunk 6]
```diff
@@ -1019,14 +1022,14 @@ trusted log that has actively decided to be malicious. It can carry
...
+the log, one way to detect this attack is to resolve each view
+of the log to the most recent STHs available and then force the log to
+present a consistency proof. (Which it cannot.) We highly recommend
```
Comment (ln5) [inline draft-ietf-trans-gossip.md:1044]: "We highly recommend" again.

File: draft-ietf-trans-gossip.md [hunk 7]
```diff
@@ -1330,7 +1330,9 @@ only client pollinating that STH cross-origin.
 
 It is mitigated partially because the log is limited in the number of
 STHs it can issue. It must 'save' one of its STHs each MMD to perform
-the attack.
+the attack. A non-compliant log, who issues additional STHs, can be
```
Comment (ln5) [inline draft-ietf-trans-gossip.md]: This has been addressed in #19.

**Review by ln5 — CHANGES_REQUESTED** [review]

File: draft-ietf-trans-gossip.md (hunk 2 above)
Comment (tomrittervg) [reply draft-ietf-trans-gossip.md]: Well, this document is specifying the mechanism and standards of gossiping, not auditing. There's lot of things an auditor should do, and we don't talk about 90% of them.

**Comment by tomrittervg** [comment]
Okay I think this is good to go. I used github conflict resolution thingie, it maybe worked?

File: draft-ietf-trans-gossip.md (hunk 2 above)
Comment (ln5) [reply draft-ietf-trans-gossip.md]: OK.

**Pull request merged**